docs/contributing.rst

```diff
@@ -7,38 +7,39 @@ Contributing
 Hacking
 =======
 
-Start by :doc:`installing dependencies and setting up Let's Encrypt
-<using>`.
+The code base, including your pull requests, **must** have 100% unit
+test coverage, pass our `integration`_ tests **and** be compliant with
+the :ref:`coding style <coding-style>`.
 
-When you're done activate the virtualenv:
 
-.. code-block:: shell
+Bootstrap
+---------
 
-   source ./venv/bin/activate
+Start by :ref:`installing Let's Encrypt prerequisites
+<prerequisites>`. Then run:
 
-This step should prepend you prompt with ``(venv)`` and save you from
-typing ``./venv/bin/...``. It is also required to run some of the
-`testing`_ tools. Virtualenv can be disabled at any time by typing
-``deactivate``. More information can be found in `virtualenv
-documentation`_.
+.. code-block:: shell
 
-Install the development packages:
+   ./bootstrap/dev/venv.sh
+   ./bootstrap/dev/venv3.sh
```

Comment:
As we are not currently developing and testing for Python 3+, I don't think this should be here.